src/main/java/core/basesyntax/strategy/OperationStrategy.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface OperationStrategy {
+    public OperationHandler chooseHandler(FruitTransaction transaction);
```

Comment:
In Java, interface methods are implicitly public, so the `public` modifier is redundant and can be removed.